README.md

```diff
@@ -170,8 +170,8 @@ List of unirec fields exported together with basic flow fields on interface by R
 | RTSP_RESPONSE_SERVER         | string | RTSP response server field  |
 | RTSP_RESPONSE_CONTENT_TYPE   | string | RTSP response content type  |
 
-### HTTPS
-List of unirec fields exported together with basic flow fields on interface by HTTPS plugin.
+### TLS
+List of unirec fields exported together with basic flow fields on interface by TLS plugin.
 
 | UniRec field        | Type   | Description                  |
 |:-------------------:|:------:|:----------------------------:|
```

Comment:
The plugin itself is renamed but output fields are the same?

Reply:
My mistake, README wasn't up to date. The fields have changed, and the README reflects that now.